modules/hooks.nix

```diff
@@ -3392,9 +3392,15 @@ lib.escapeShellArgs (lib.concatMap (ext: [ "--ghc-opt" "-X${ext}" ]) hooks.ormol
           entry =
             let
               inherit (hooks.statix) package settings;
+              mkOptionName = k:
+                if builtins.stringLength k == 1
+                then "-${k}"
+                else "--${k}";
```

Comment:
Why are we re-implementing the default behaviour: https://github.com/NixOS/nixpkgs/blob/nixos-24.05/lib/cli.nix#L67-L73